src/preamble.js

```diff
@@ -136,10 +136,17 @@ var JSfuncs = {
       stringToUTF8(str, ret, len);
     }
     return ret;
+  },
+  'boolToC': function (bool) {
```

Comment:
Thanks for the PR!

I don't think I understand it yet. Do we need `boolToC`? For strings, we convert them from native JS objects into something C can handle, but for bools I think things should just work, a JS boolean value will be `|0`-ed when we send it to C anyhow, and that will correctly convert it, since `(true | 0) === 1` and `(false | 0) === 0` (and 0/1 are the values C expects).

Also, please add to the existing ccall test (`test_ccall` in `tests/test_core.py`). In particular, if `boolToC` is needed, let's make sure the test shows a case where it is.

Reply:
Okay, yeah now that I'm thinking about it, I don't think it's necessary but if it is I can add it.